### DISCUSSION TITLE
英文标题没有全大写

### DISCUSSION THREAD

**Discussion by xskxzr** [tuna/thuthesis#616] [Q&A] [answered]
按要求英文论文好像要求标题要全大写
<img width="905" alt="截屏2021-03-22 上午9 14 03" src="https://user-images.githubusercontent.com/7037227/111928576-ff990500-8aee-11eb-966d-76270d038b17.png">

<img width="814" alt="截屏2021-03-22 上午9 15 33" src="https://user-images.githubusercontent.com/7037227/111928636-30793a00-8aef-11eb-9619-96c8fb0a17b3.png">

但模板生成的标题只有首字母大写了
<img width="883" alt="截屏2021-03-22 上午9 16 42" src="https://user-images.githubusercontent.com/7037227/111928706-5acaf780-8aef-11eb-8326-af4b2ec3203f.png">

**Comment by zepinglee** [comment, marked as answer]
检查以下你的版本？

**Reply by xskxzr** [reply]
原来最近有更新啊，我一直用的7.1.0，更新只需要覆盖thuthesis.cls吗？

**Reply by Harry-Chen** [reply]
是的，从 7.1.0 升级覆盖 cls 即可，别的没有改动

**Reply by yaox12** [reply]
> 原来最近有更新啊，我一直用的7.1.0，更新只需要覆盖thuthesis.cls吗？

貌似还要更新`.bst`，虽然我没具体看改了啥
